torchvision/io/_video_opt.py

```diff
@@ -24,21 +24,20 @@
 # simple class for torch scripting
 # the complex Fraction class from fractions module is not scriptable
 class Timebase(object):
-    __annotations__ = {"numerator": int, "denominator": int}
-    __slots__ = ["numerator", "denominator"]
+    __annotations__: Dict[str, Type[int]] = {"numerator": int, "denominator": int}
```

Comment:
Shouldn't that be

```suggestion
    __annotations__: Dict[str, Type] = {"numerator": int, "denominator": int}
```

Otherwise we can only have integer attributes, right?

Reply:
Correct, but I thought this was the purpose of the annotation attribute :man_shrugging: